webapp/components/member_list_team.jsx

```diff
@@ -122,7 +122,6 @@ export default class MemberListTeam extends React.Component {
 
         return (
             <SearchableUserList
-                style={this.props.style}
```

Comment:
Thanks for changing those, but you missed one more. The style prop also needs to be removed from MemberListTeam's propTypes definition and the TeamMembersModal that uses this